src/Particular.PlatformSample/Particular.PlatformSample.csproj

```diff
@@ -15,7 +15,7 @@
     <PackageReference Include="Particular.Packaging" Version="0.2.1" PrivateAssets="all" />
     <PackageReference Include="Particular.PlatformSample.ServiceControl" Version="3.7.0" PrivateAssets="none" />
     <PackageReference Include="Particular.PlatformSample.ServiceControl.Monitoring" Version="2.3.0" PrivateAssets="none" />
-    <PackageReference Include="Particular.PlatformSample.ServicePulse" Version="1.19.0" PrivateAssets="none" />
+    <PackageReference Include="Particular.PlatformSample.ServicePulse" Version="1.19.1-pullrequest0746-0089" PrivateAssets="none" />
```

Comment:
Won't the real package (and version of ServicePulse) be a new minor?

Reply:
it'll be `1.20`

Reply:
@DavidBoike I'll swap out the pre-release with the RTM package once it is officially released.